foundation-extensions/identity-for-edge-network/api-reference.md

```diff
@@ -1,6 +1,6 @@
 # API Reference
 
-## extensionVersion
+## extensionVersion <a id="extensionversion"></a>
```

Comment:
nit: Not strictly necessary. Headers are automatically converted to links by GitBook. The anchor name is the same as the header, lowercased and with special characters (such as a space) converted to "-".

Reply:
I  saw we have anchor for consent and Edge, so I added that for Edge Identity.  If we don't need that, should I remove the anchors for Consent and Edge instead?

Reply:
Yes, we should be able to safely remove all the anchors that have the same name as the header. For the ones that are different than header name, check if there are any links that need to be updated too.